report/4.Design/Object_Localization.tex

```diff
@@ -0,0 +1,113 @@
+\section{Object localization}\label{Design:ObjectLocalization}
+An early working product was favored in order to get a baseline for all components to be further developed from.
```

Comment:
I think the final part of the sentence needs to be changed. I think something like: (VERY MUCH SUGGESTION)
```suggestion
An early working product was favored in order to, as it would allow us to have a baseline product that we can further iterate on.
```